docs/contributing/building/environment.md

```diff
@@ -14,72 +14,44 @@
     If you want to build tracee on your local machine
     [read this](./building.md).
 
+## Quick steps
 
-## Quick steps (**impatient readers**)
+1. Build tracee environment:
 
-!!! Example
-
-    * Build and execute **tracee**:
-    
-        ```console
-        make -f builder/Makefile.tracee-make alpine-prepare
-        make -f builder/Makefile.tracee-make alpine-shell
-        ```
-
-        and inside the container:
-
-        ```console
-        make clean
-        make tracee
-        sudo ./dist/tracee \
-            -o option:parse-arguments \
-            --scope comm=bash \
-            --scope follow
-        ```
-    
-    Now, in your host's bash shell, execute a command. You will see all events
-    (except scheduler ones) being printed, in "table format", to stdout.
-    
-    * Build and execute **tracee**:
-    
-        ```console
-        make -f builder/Makefile.tracee-make alpine-prepare
-        make -f builder/Makefile.tracee-make alpine-shell
-        ```
+    ```bash
+    make -f builder/Makefile.tracee-make alpine-prepare
+    make -f builder/Makefile.tracee-make alpine-shell
+    ```
 
-        and inside the container:
+2. Build and execute tracee:
 
-        ```console
-        make clean
-        make all
-        sudo ./dist/tracee \
-            -o format:json \
-            -o option:parse-arguments \
-            --scope comm=bash \
-            --scope follow 
-        ```
-    
-    Now, in your host's bash shell, execute: `sudo strace /bin/ls` and observe
-    tracee warning you about a possible risk (with its Anti-Debugging signature).
+    ```bash
+    make clean
+    make tracee
+    sudo ./dist/tracee \
+        -o option:parse-arguments \
+        --scope comm=bash \
+        --scope follow
+    ```
 
-Now, for **more patient readers** ...
+Now, in your host's shell, execute a command. You will see all events
+(except scheduler ones) being printed, in "table format", to stdout.
 
 ## How to build and use the environment
 
 In order to have a controlled building environment for tracee, tracee provides
-a `Makefile.tracee-make` file that allows you to create and use a docker
-container environment to build & test **tracee**. 
+a `Makefile.tracee-make` file that allows you to create and use a docker container environment to build & test **tracee**.
+
+There are Two different environments that are maintained for building tracee:
 
-Two different environments are maintained for building tracee:
+* **Alpine**
+* **Ubuntu**
 
-* Alpine
-* Ubuntu
+The reason for that is that **Alpine Linux** is based in the [musl](https://en.wikipedia.org/wiki/Musl) C standard library.
```

Comment:
I think we can combine both:

The reason for that is that **Alpine Linux** is based in the [musl](https://en.wikipedia.org/wiki/Musl) C standard library, while the **Ubuntu Linux** uses [glibc](https://en.wikipedia.org/wiki/Glibc).